omnibus/config/software/datadog-agent-integrations-py2.rb

```diff
@@ -11,6 +11,8 @@
 dependency 'datadog-agent'
 dependency 'pip2'
 
+# Include traps db file in snmp.d/traps_db/
+dependency 'snmp-traps'
```

Comment:
Should we add the dependency once to https://github.com/DataDog/datadog-agent/blob/5bba7214704fc6e6c7c8a124a00afe0275c504ff/omnibus/config/software/datadog-agent.rb instead since the feature is for a corecheck, not an snmp integration ?

Reply:
I'd be interested to have agent-platform opinion on this, I was a bit lost with the different files used and wasn't sure where was the best place.
Can be an option indeed but `datadog-agent.rb` does not seem to define much dependencies at the moment